app/templates/account.hbs

```diff
@@ -1,3 +1,8 @@
+<div class="ui stackable grid">
+  <div class="sixteen wide column">
+    {{unverified-user-message}}
+  </div>
+</div>
 <div class="row">
   <div class="sixteen wide column">
     {{#tabbed-navigation}}
```

Comment:
Can you add the message here ? Before {{Tabbed-navigation}}

Reply:
on the other pages it is like this as implemented .